src/rtctools_heat_network/pycml/component_library/heat/ates.py

```diff
@@ -60,13 +60,13 @@ def __init__(self, name, **modifiers):
 
         # Heat stored in the ates
```

Comment:
Maybe add general comment that Heat_ates is negative when discharging ATES. Or note that ATES has been implemented as demand connection

Reply:
comment added in HEX branch